civihr_default_theme/scss/base/_task-assignments-modal-temp.scss

```diff
@@ -15,8 +15,7 @@ fieldset {
 }
 
 // Opened Modal
```

Comment:
can we update the doc from `// Opened Modal` to something like `SSP Upload Document`? or even if we removed it, its fine